src/api/v1/api.json

```diff
@@ -494,7 +494,7 @@
           "isImplemented": false,
           "auth": true,
           "intendedFor": ["ORG_ADMIN"],
-          "scope": ["organisations_feedback-parent:write"],
+          "scope": ["organisations_feedback-child:write"],
```

Comment:
Why has this been changed to child? Should it not be own?

Reply:
Yes it should be `own` 👍